docs/install/kubernetes.md

```diff
@@ -222,7 +224,36 @@ coder:
 1. Lastly, your PG connection URL will look like:
 
 ```shell
-postgres://<user>:<password>@databasehost:<port>/<db-name>?sslmode=require&sslcert=$HOME/.postgresql/postgres.crt&sslkey=$HOME/.postgresql/postgres.key"
+postgres://<user>:<password>@databasehost:<port>/<db-name>?sslmode=require&sslcert="$HOME/.postgresql/postgres.crt&sslkey=$HOME/.postgresql/postgres.key"
```

Comment:
@ericpaulsen, as this PG verification step is common to all installation methods, should it be refactored to a new section (advanced install)? It does not fit here in K8s.

Also, We should probably create a generic guide on creating a k8s secret and mounting it as a volume in the k8s pod and then refer to it within docs whenever we discuss mounting any k8s secrets. 

cc: @coder/pms

Reply:
@matifali i agree - i can move this to a guide.